### ISSUE TITLE
Master branch IMDB lookup is not working

### ISSUE THREAD

**Issue by crunchygremlin** [spotweb/spotweb#323]
this may be a known issue but from what i can tell the master branch imdb lookup is not working. 
TLDR.
To fix this
i backed up my lib/services/signing directory.
i copied the dev lib/services folder to my spotweb install location.
then copied back the signing directory.
Now IMDB lookup works great. 
as the db schema has changed between master and dev just copying over the lib directory is not likely to work. did not work for me.
I also copied from dev lib/page and dev lib/dto
not sure if they are needed or not. likely doesnt matter. 

http://myserver/spotweb/index.php?page=newznabapi&imdbid=4425200&t=m&extended=1&movie&apikey=myapikey
was returning no results for me. 
although removing the imdbid and using t=t and specifying a cat that would contain what i wanted and a q= i did get back the expected results. 
searching for a title this was also worked. But i could not figure out how to get the calling app to plug in the title it was looking for. 

I did a whole bunch of stuff trouble shooting this. like 20 hours or so of trying to get spotweb working. It
I found that anytime i used t=m and an imdbid i got nothing back. no results.

i also found that tvmazeid did not work and it still doesn't work.
all that are returned are the current t=100 results when doing a tvmaze lookup. 
if what im looking for happens to be in that list sickrage is happy.
if not it doesnt find it. 
so this...
http://myserver/spotweb/index.php?page=newznabapi&t=t&tvmazeid=74280&extended=1&apikey=myapikey
returns the top 100 most recent tv category results. 

By doggedly trying to figure out why i was getting no results even though i could see what i wanted was in the list i worked through the code and almost started to get php debugging working before i tried some other stuff. 

Ok please forgive any ignorance here. I have been crash coursing on newsnabapi and spotweb php.

**Comment by mesa57** [comment]
Except for the read,me file, there is currently no difference betwen master and develop.

**Comment by crunchygremlin** [comment]
That can't be.
I installed the master.
Unless I somehow got an older commit.

That would really burn my ass

**Comment by crunchygremlin** [comment]
i was following this guide.
https://github.com/spotweb/spotweb/wiki/Install-Spotweb-on-Ubuntu-12.04-LTS-(Precise-Pangolin)-(english)
the tarball referenced there must be out of date.

**Comment by mesa57** [comment]
The tarball is prehistoric.

**Comment by mesa57** [comment]
Should be updated now to 1.3.3

**Comment by crunchygremlin** [comment]
Alright. Well at least all that time wasn't completely wasted.
Thanks for help.
Now to install the real deal.
And get past all these db issues
